packages/web/src/components/organisms/EntityDetails/EntityDetailsHeader.tsx

```diff
@@ -75,8 +77,8 @@ const EntityDetailsHeader: FC<EntityDetailsHeaderProps> = ({
             type="warning"
```

Comment:
This tag should be also visible when `!isAgentAvailable`, even when `!outOfSync`